senza/cli.py

```diff
@@ -952,9 +952,13 @@ def status(stack_ref, region, output, w, watch):
                         except:
                             http_status = 'ERROR'
                     else:
-                        try:
-                            answers = dns.resolver.query(name, 'CNAME')
-                        except:
+                        for record_type in ["CNAME", "A", "AAAA"]:
```

Comment:
@hjacobs I reverted that change and I'll create a Pull Request later to fix this (maybe by checking the route53 api).